_codux/boards/card-collection.board.css

```diff
@@ -0,0 +1,100 @@
+.cardCollection {
+    display: flex;
+    padding: 48px;
+    flex-direction: column;
+    align-items: center;
+    font-family: Arial, sans-serif;
```

Comment:
What's the reason for this? It will override any font that is used by the user for this snippet's container.